TodoList/sources/TodoList.Repository/Helpers/ConnectionHelper.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Configuration;
+
+namespace TodoList.Repository.Helpers
+{
+    internal static class ConnectionHelper
+    {
+        internal static string GetDbConnection()
```

Comment:
* Make it a lambda, definitely
* Create a contract for it, the reason this class works is that you always run in from `WebAPI`. It would be much better to leverage `inversion of control` (or `dependency injection` more precisely) to define an interface, much like [`options pattern`](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/configuration?tabs=basicconfiguration#using-options-and-configuration-objects) in .Net Core. This contract will have its implementation in `API` layer while `repository` will require an instance fulfilling the contract in its constructor.
* Move the call of `ConfigurationManager` to constructor – it is OK to cache it at least per request.